Lib/platform.py

```diff
@@ -189,22 +189,28 @@ def libc_ver(executable=None, lib='', version='', chunksize=16384):
             # sys.executable is not set.
             return lib, version
 
-    libc_search = re.compile(b'(__libc_init)'
-                          b'|'
-                          b'(GLIBC_([0-9.]+))'
-                          b'|'
-                          br'(libc(_\w+)?\.so(?:\.(\d[0-9.]*))?)', re.ASCII)
+    libc_search = re.compile(
+        br'(__libc_init)'
+        br'|'
+        br'(GLIBC_([0-9.]+))'
+        br'|'
+        br'(libc(_\w+)?\.so(?:\.(\d[0-9.]*))?)'
+        br'|'
+        br'(musl-([0-9.]+))'
+        br'',
+        re.ASCII)
 
     V = _comparable_version
     # We use os.path.realpath()
     # here to work around problems with Cygwin not being
     # able to open symlinks for reading
     executable = os.path.realpath(executable)
+    ver = None
     with open(executable, 'rb') as f:
         binary = f.read(chunksize)
         pos = 0
         while pos < len(binary):
-            if b'libc' in binary or b'GLIBC' in binary:
+            if b'libc' in binary or b'GLIBC' or 'musl' in binary:
```

Comment:
```suggestion
            if b'libc' in binary or b'GLIBC' or b'musl' in binary:
```

Reply:
Oh, that's a nasty one.  The actual fix is `b'GLIBC` in binary or b'musl' in binary`.  That's why it passed without the b...it just ran slower ;)  (A lot slower, somewhat to my surprise.)